src/BAE910.cpp

```diff
@@ -0,0 +1,109 @@
+#include "BAE910.h"
+
+BAE910::BAE910(uint8_t ID1, uint8_t ID2, uint8_t ID3, uint8_t ID4, uint8_t ID5, uint8_t ID6, uint8_t ID7) : OneWireItem(ID1, ID2, ID3, ID4, ID5, ID6, ID7)
+{
+    for (uint8_t i = 0; i < 0x80; ++i)
+        memory.bytes[i] = 0x00;
```

Comment:
could be replaced by the much faster memset(), i included a version that does basically the same if a compiler for a chip (attiny) is missing this function
--> https://github.com/orgua/OneWireHub/blob/master/src/platform.h @ the end